src/Index.js

```diff
@@ -812,6 +813,10 @@ Index.prototype.saveRule = function(rule, opts, callback) {
     opts = {};
   }
 
+  if (rule.objectID === undefined || rule.objectID === '') {
+    throw new Error('Missing or empty objectID field for rule');
```

Comment:
We can simplify the condition to ˋ!rule.objectID` since it should always be a string the double check is a bit redundant. We also miss the ˋnull` case.

Reply:
I've changed the condition to `!objectID || objectID.length === 0`. WDYT?